### PR TITLE
adding one line to each file

### PR DESCRIPTION
**Pull request by ManolyaTam**
please add these changes to main branch

### CODE REVIEW

**Review by ManolyaTam — COMMENTED** [review]
hello, asia!

**Comment by ManolyaTam** [comment]
@AsiaShalaldeh 
the person who made the pull request cannot merge ...

File: colors.txt [hunk 1]
```diff
@@ -2,3 +2,4 @@ red
 white
 black
 green
+yellow
```
Comment (AsiaShalaldeh) [inline colors.txt:5]: it should be yellowwwwwwwwww

Comment (ManolyaTam) [reply colors.txt:5]: LOL

File: letters.txt [hunk 2]
```diff
@@ -3,3 +3,4 @@ B
 C
 D
 E
+F
```
Comment (AsiaShalaldeh) [inline letters.txt:6]: Nooooo, it's Z

File: colors.txt (hunk 1 above)
Comment (ManolyaTam) [reply colors.txt:5]: just merge it😂😂😂

File: numbers.txt [hunk 3]
```diff
@@ -1,6 +1,7 @@
+0
 1
 2
 3
 4
 5
-6
+6
```
Comment (AsiaShalaldeh) [inline numbers.txt:7]: Why did you delete my line??????????????????

**Review by AsiaShalaldeh — CHANGES_REQUESTED** [review]

File: numbers.txt (hunk 3 above)
Comment (ManolyaTam) [reply numbers.txt:7]: i did not 
i just inserted the 0 above all the lines
